frontend/src/components/Mobile/InputField.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { Dispatch, SetStateAction, useState } from 'react'
+import styled from 'styled-components'
+
+import { Input } from 'antd'
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin: 10px 0px;
+`
+
+const StyledInput = styled(Input)<{ haserror?: boolean }>`
+  &.ant-input {
+    background: #f3f3f9;
+    width: 100%;
+    border-radius: 30px;
+    border: 1px solid ${(props) => (props.haserror ? 'red' : '#f3f3f9')};
+    padding: 5px 10px;
+    margin: 0px 0px 0px 0px;
+    ${(props) => props.haserror && `background: #ffd1d1;`}
+  }
+  &.ant-input::placeholder {
+    color: #bfbfbf;
+    font-size: 0.8rem;
+    padding-left: 5px;
+  }
+`
+
+const { TextArea } = Input
+const StyledTextArea = styled(TextArea)<{ haserror?: boolean }>`
+  &.ant-input {
+    background: #f3f3f9;
+    width: 100%;
+    border-radius: 20px;
+    border: 1px solid ${(props) => (props.haserror ? 'red' : '#f3f3f9')};
+    padding: 5px 10px;
+    margin: 0px 0px 0px 0px;
```

Comment:
if the styling is very similar consider abstracting out using styled-components' css